src/Evaluators/reduced_evaluator.jl

```diff
@@ -86,6 +86,43 @@ function ReducedSpaceEvaluator(
     )
 end
 
+function ReducedSpaceEvaluator(nlp::ReducedSpaceEvaluator, device = nothing) 
+    if device == CUDADevice()
+        VT = CuVector 
+        MT = CuMatrix
+    elseif device == CPU()
+        buffer = PolarNetworkState(nlp.buffer, device)
```

Comment:
Do we really need to instantiate the `buffer` here? I am wondering whether this line is redundant with line 101.

Reply:
That's a mistake :-)